src/bin/index.ts

```diff
@@ -61,6 +64,12 @@ export async function bin(args: string[]) {
 
 	if (code) {
 		logLine();
+
+		if (zodError) {
+			const validationError = fromZodError(zodError);
+			prompts.outro(chalk.red(validationError));
```

Comment:
> `prompts.outro`

[Bug] The terminal output looks a little buggy with two outros:

```plaintext
┌  Welcome to create-typescript-app ! 🎉
│
│  ⚠️ This template is early stage, opinionated, and not endorsed by the TypeScript team. ⚠️
│  ⚠️ If any tooling it sets displeases you, you can always remove that portion manually. ⚠️
│
●  Tip: to run again with the same input values, use: npx create-typescript-app --mode create --base everything --email wat
│
│
└  Validation error: Invalid email at "email"

└  Operation cancelled. Exiting - maybe another time? 👋
```

How about switching to a `logLine`? I think the ideal would be a blank line around the validatione rror:

```plaintext
┌  Welcome to create-typescript-app ! 🎉
│
│  ⚠️ This template is early stage, opinionated, and not endorsed by the TypeScript team. ⚠️
│  ⚠️ If any tooling it sets displeases you, you can always remove that portion manually. ⚠️
│
●  Tip: to run again with the same input values, use: npx create-typescript-app --mode create --base everything --email wat
│
│  Validation error: Invalid email at "email"
│
└  Operation cancelled. Exiting - maybe another time? 👋
```